examples/speech_to_text/visualize.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import argparse
+import pprint
+
+
+def read_scores_from_folder(folder_path):
+    score_file_path = os.path.join(folder_path, "scores.tsv")
```

Comment:
Why is `scores.tsv` hardcoded? Is this a standard name or should we make it a parameter?

Reply:
It is a standard name. Whenever you run Simuleval, a file is created to store the score of the evaluation and the file is usually saved as scores.tsv